tests/ProductBundle/Seo/Services/FacebookTest.php

```diff
@@ -63,4 +65,65 @@ public function testAlterPage(): void
 
         $this->assertContains('O-some product', $content);
     }
+
+    public function testAlterPageImage()
+    {
+        $mediaPool = $this->createMock(Pool::class);
+        $seoPage = new SeoPage('test');
+        $extension = new SeoExtension($seoPage, 'UTF-8');
+        $numberHelper = $this->createMock(NumberHelper::class);
+        $currencyDetector = $this->createMock(CurrencyDetectorInterface::class);
+        $product = new ProductFbMock();
+        $router = $this->createMock(RouterInterface::class);
+
+        //Prepare currency
+        $currency = new Currency();
+        $currency->setLabel('EUR');
+        $currencyDetector->expects($this->any())
+                ->method('getCurrency')
+                ->willReturn($currency);
+
+        $imageDummyLink = '/upload/dummy.png';
+
+        // Test getImage
+        $image = $this->createMock(MediaInterface::class);
+
+        $imageProvider = $this->createMock(ImageProvider::class);
+        $imageProvider->expects($this->any())
+            ->method('generatePublicUrl')->willReturn($imageDummyLink);
+
+        $image->expects($this->any())
+            ->method('getName')->willReturn('correctMedia');
+        $image->expects($this->any())
+            ->method('getWidth')->willReturn(1111);
+        $image->expects($this->any())
+            ->method('getHeight')->willReturn(2222);
+        $image->expects($this->any())
+            ->method('getProviderName')->willReturn($imageProvider);
+        $image->expects($this->any())
+            ->method('getContentType')->willReturn('image/png');
+
+        $mediaPool->expects($this->any())
+            ->method('getProvider')->willReturn($imageProvider);
+
+        $product->setImage($image);
+
+        // Check if the header data are correctly registered
+        $fbService = new Facebook($router, $mediaPool, $numberHelper, $currencyDetector, 'test', 'test', 'reference');
+        $fbService->alterPage($seoPage, $product);
+        $content = $extension->getHeadAttributes();
+
+        $this->assertContains('fb: http://ogp.me/ns/fb#', $content);
+
+        $content = $extension->getMetadatas();
+
+        // image link
+        $this->assertContains('test'.$imageDummyLink, $content);
```

Comment:
@core23 test for domain + image path

Reply:
Can you please use something more relastic value for a domain? This will help if someone else will work on this code fragment later.

Reply:
Ok, undestand i used that was already done in the previous test